lib/streamlit/elements/widgets/button.py

```diff
@@ -873,11 +873,12 @@ def _page_link(
             for page_data in all_app_pages.values():
                 full_path = page_data["script_path"]
                 page_name = page_data["page_name"]
+                url_pathname = page_data["url_pathname"]
                 if requested_page == full_path:
                     if label is None:
-                        page_link_proto.label = page_name.replace("_", " ")
+                        page_link_proto.label = page_name
```

Comment:
Why don't we have to replace underscores anymore now? Just want to make sure we capture a little bit of context here in the comments in case we have to reason about it again in the future